File: FreakFightsFan.Shared/Features/Events/Responses/EventDto.cs
using FreakFightsFan.Shared.Features.DictionaryItems.Responses;

namespace FreakFightsFan.Shared.Features.Events.Responses;

public class EventDto
{
    public int Id { get; init; }
    public DateTime Created { get; set; }
    public DateTime Modified { get; set; }

    public string Name { get; set; }
    public DateTime Date { get; set; }
    public int FederationId { get; set; }
    public MyDictionaryItemDto? City { get; set; }
    public MyDictionaryItemDto? Hall { get; set; }
}

File: freak-fights-fan/shared/features/events/responses/EventDto.ts
import { MyDictionaryItemDto } from "../../dictionaryItems/responses/MyDictionaryItemDto";

export interface EventDto {
    id: number;
    created: Date;
    modified: Date;

    name: string;
    date: Date;
    federationId: number;
    city: MyDictionaryItemDto | undefined;
    hall: MyDictionaryItemDto | undefined;
}